src/main/resources/templates/macros/report/docstring.vm

```diff
@@ -8,7 +8,7 @@
           <a>Doc string</a>
       </div>
       <div id="msg-$docStringId" class="collapse collapsable-details">
-        <pre>$docString.getValue()</pre>
+        <pre>$escHtml.translate($docString.getValue())</pre>
```

Comment:
I don't understand. If you write a gherkin scenario and you put an html in the doc string, i assume is what you want to see in the report. For example:

Given the next html
'''
<b>Bold Text</b>
'''
When I open it in a browser
Then I should see the text "Bold Text" render as bold

When I see the report, I would like to see the raw text